src/js/view/preview.ts

```diff
@@ -1,17 +1,23 @@
 import { style } from 'utils/css';
+import type Model from 'controller/model';
```

Comment:
view-model or player-view-model?

If there is _**any**_ place where we are directly using "the" player model in src/view/ it should be called out in a comment, because we want the view-model and it's player-view-model to play go-between and prevent that.

Reply:
this is good to know - we are passing in the view model, just verified in the trace